examples/systemcalls/systemcalls.c

```diff
@@ -59,9 +74,35 @@ bool do_exec(int count, ...)
  *
 */
 
+    pid_t pid = fork();
+
+    if(pid == -1)
+    {
+	return false;
+    }
+	
+    if(pid == 0)
+    {
+	execv(command[0], &command[0]);
+	return false;
```

Comment:
@ECleverito I'm not sure return(false) will do what you want here, I think you need to exit(-1) so you can notify the parent process there was an error